packages/jellyfish-transaction/src/script/dftx/dftx.ts

```diff
@@ -177,6 +177,8 @@ export class CDfTx extends ComposableBuffer<DfTx<any>> {
         return compose<ICXSubmitDFCHTLC>(CICXSubmitDFCHTLC.OP_NAME, d => new CICXSubmitDFCHTLC(d))
       case CCreateLoanScheme.OP_CODE:
         return compose<CreateLoanScheme>(CCreateLoanScheme.OP_NAME, d => new CCreateLoanScheme(d))
+      case CICXSubmitEXTHTLC.OP_CODE:
```

Comment:
CreateLoanScheme should be arranged after this

Reply:
I think it's okay🤔 , the order might not be perfect since multiple PRs with different scope can get merged to the `main` concurrently.

Reply:
Yeah @surangap +1

It's hard to arrange when multiple PR is being slated and for review and merging. I usually refactor afterward when it's done. The alternative is breaking down `dftx.ts` into smaller sub modular sections, but it's beyond scope for now.